e2e-testing/cypress/locators/ChangeContactEmailAddressLocators.js

```diff
@@ -0,0 +1,8 @@
+const ChangeContactEmailAddressLocators = {
+    settingsBtn: '[data-qa="btn-settings"]',
+    email: '[data-qa="email"]',
```

Comment:
Minor change, rename this to emailField just so that we can tell it's a field being targeted. May be a bit confusing to see two email (this and line 5) so let's have a distinguish.

Reply:
Looks good. Just this one minor change and it should be good to go.